requirements-dev.txt

```diff
@@ -21,5 +21,5 @@ databroker
 ophyd
 scipy
 pre-commit
-pytest-docker==1.0.1
-databroker @ git+https://github.com/bluesky/databroker.git@v2.0.0b13#egg=databroker
+databroker @ git+https://github.com/bluesky/databroker.git@v2.0.0b38#egg=databroker
+tiled[all]
```

Comment:
Does it definitely need the full installation?

Reply:
Added https://github.com/bluesky/bluesky-adaptive/issues/38